images/bootstrap/bootstrap.sh

```diff
@@ -48,12 +48,13 @@ then
 	        --from-file=cassandra-node.pem=cassandra-node.pem
 fi
 
-pithosctl init
-
 if [ $(/opt/bin/kubectl get nodes -l pithos-role=node -o name | wc -l) -ge 3 ]
 then
-    /opt/bin/kubectl scale statefulset cassandra --replicas=3
+    sed -i 's/replicas: 1/replicas: 3/' /var/lib/gravity/resources/cassandra.yaml
```

Comment:
Can you provide some background for this change?
What I usually do when I think a place needs a bit of explanation but comments would not be acceptable or it makes sense only in context of this one PR - I review the PR myself leaving comments where I deem appropriate.

Reply:
I tried to fix the problem with the postInstall hook. Installation hook finishes before all 3 cassandra nodes up and post installation hook fails after 6 tries.

Reply:
Unfortunately, this fix does not help at all because bootstrap script is written with rigging function which does not check resource status after usage. Let's say it will be a temporary change here because I am thinking about migration to helm.